### PR TITLE
Make FeedSpider 2 a PWA

### PR DESCRIPTION
**Pull request by codepoet80**
The following changes allow FeedSpider to work as a modern Progress Web App (PWA) on iOS, Android and Chrome...

- Bumped version, added manifest.json and stub service worker. Added appropriate icon references in manifest and index.html. Added metatags for use on iOS.
- Added stub serviceworker so Chrome will offer the "Install" option
- Moved InoReader to a (now public) nginx reverse proxy to allow CORS, so that it will load on a modern browser. Other back-ends not yet supported (except on older devices that don't care about CORS). Example nginx config file included.
- Other back-ends to be supported in future PRs.

Try it out at [https://feedspider.online/app](https://feedspider.online/app)

### CODE REVIEW

File: enyo-app/index.html [hunk 1]
```diff
@@ -16,7 +38,47 @@
...
+		<script>
+			//Register service worker (stub)
+			if ('serviceWorker' in navigator) {
+				window.addEventListener('load', function() {
+					navigator.serviceWorker.register('serviceworker.js').then(function(registration) {
+					// Registration was successful
+					console.log('ServiceWorker registration successful with scope: ', registration.scope);
+					}, function(err) {
+					// registration failed :(
+					console.log('ServiceWorker registration failed: ', err);
```
Comment (OthelloVentures) [inline enyo-app/index.html]: console.log should be avoided in the production app

File: enyo-app/index.html [hunk 2]
```diff
@@ -16,7 +38,47 @@
...
+					});
+				});
+			}
+
+			//Stop iOS bouncy overscroll
+			function onTouchStart(e) {
+				// Save position of touch
+				console.log("touchstart");
```
Comment (OthelloVentures) [inline enyo-app/index.html]: As above

File: enyo-app/index.html [hunk 3]
```diff
@@ -16,7 +38,47 @@
...
+				window.lastY = touch.pageY;
+			}
+
+			function onTouchMove(e) {
+				console.log("touchmove");
+				// Check user isn't scrolling past content. If so, cancel move to prevent ios bouncing
+				const touch = e.touches[0] || e.changedTouches[0];
+				y = touch.pageY;
+				if (y < window.lastY && e.srcElement.scrollTop == (e.srcElement.scrollHeight - e.srcElement.clientHeight)) {
+					console.log("user is trying to scroll down without anywhere to scroll to. Canceling propagation.");
```
Comment (OthelloVentures) [inline enyo-app/index.html]: As above

File: enyo-app/index.html [hunk 4]
```diff
@@ -16,7 +38,47 @@
...
+					e.preventDefault();
+				} else if (y > window.lastY && e.srcElement.scrollTop == 0) {
+					console.log("user is trying to scroll up without anywhere to scroll to. Canceling propagation.");
```
Comment (OthelloVentures) [inline enyo-app/index.html]: As above

**Review by OthelloVentures — CHANGES_REQUESTED** [review]

**Comment by codepoet80** [comment]
I don't know how to revise a PR to make those changes, so I'll close this PR and submit a new one with some other modifications we're making for newer LuneOS support.

**Pull request closed without merging**